vivisect/qt/funcgraph.py

```diff
@@ -503,6 +505,13 @@ def _renderMemory(self):
                 self.mem_canvas.addText('Invalid Address: %s (%s)' % (expr, e))
                 return
 
+            # get a location anchor if one exists, otherwise, we may end up in no-man's land,
+            # since we rely on labels, which only exist for the base of a location.
+            loc = self.vw.getLocation(addr)
+            if loc is not None:
+                addr = loc[L_VA]
```

Comment:
add an else case here to return and aside from that, I think we're good.

Reply:
by moving the getLocation() into _renderMemory(), we don't want to return if there's no location... this is actually ideal, since we want to render with the addr given if there is no location, and update it to the base of the location if there is one.  because the renderer tags individual bytes if they aren't a part of a location.  this is good.  please review this part again with that understanding and see if you still want an else: case.

we only want to update addr if there is a location because the renderer, but leave it as is if there isn't a location.